src/server/main.ts

```diff
@@ -99,12 +116,26 @@ connection.onCompletion(async (textDocumentPosition) => {
 });
 
 
-connection.onRequest('aurelia-view-information', (filePath: string) => {
+connection.onRequest('aurelia-view-information', async (filePath: string) => {
+  let fileProcessor = new ProcessFiles();
+  await fileProcessor.processPath();
+  aureliaApplication.components = fileProcessor.components;

```

Comment:
Additionally, to processing files on extension startup.
We need to update `aureliaApplication.components` for new 'gotos'.

This is the fastest way I could provide updates